src/ApiClient/models/InternalComms/InternalError.ts

```diff
@@ -145,13 +145,12 @@ export default class InternalError<T extends ErrorCode = ErrorCode> {
         );
         debuginfo = debuginfo.replace(/"username":.".+?"/g, '"username": "*******"');
         debuginfo = debuginfo.replace(/"password":.".+?"/g, '"password": "*******"');
-        if (AuthController.isTokenValid()) {
-            debuginfo = replaceAll(
-                debuginfo,
-                AuthController.getTokenUnsafe()!.bearer,
-                "**************"
-            );
-        }
+
+        debuginfo = replaceAll(
+            debuginfo,
+            AuthController.getTokenUnsafe()!.bearer,
```

Comment:
Its incorrect to null assert here, please add a check that there is a token, we don't care if its valid or not but it does it need to exist if we are going to be accessing a property on it